src/main/java/com/ts/tserp/controller/BatchAnnouncementController.java

```diff
@@ -0,0 +1,33 @@
+package com.ts.tserp.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.ts.tserp.model.BatchAnnouncement;
+import com.ts.tserp.service.BatchAnnouncementService;
+
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequestMapping("/api")
+@RequiredArgsConstructor
+public class BatchAnnouncementController {
+
+	@Autowired
+	private BatchAnnouncementService batchAnnouncementService;
+	
+	@PostMapping("/add_batch")
```

Comment:
change the API name to "/add_batch_announcement"
Do the same changes for other batch announcement API's

Reply:
changes done